packages/react/src/Label/Label.tsx

```diff
@@ -8,13 +8,27 @@ import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { ForwardedRef, LabelHTMLAttributes, PropsWithChildren } from 'react'
 
+export type LabelProps = PropsWithChildren<LabelHTMLAttributes<HTMLLabelElement>> & {
+  /** Text to inform the user that the field is explicitely not required. */
+  notRequiredLabel?: string
+  /** Whether the field set is required. */
+  required?: boolean
+}
+
+export const LabelOptionalFlag = ({ notRequiredLabel }: LabelProps) => (
+  <span className="ams-label__not-required">
+    <span className="ams-visually-hidden">,</span> ({notRequiredLabel})
+  </span>
+)
+
 export const Label = forwardRef(
   (
-    { children, className, ...restProps }: PropsWithChildren<LabelHTMLAttributes<HTMLLabelElement>>,
+    { children, className, notRequiredLabel = 'niet verplicht', required, ...restProps }: LabelProps,
```

Comment:
Do we need 2 props for this? Couldn't we just use `notRequiredLabel` without a default string, which is shown when it's defined? The prop `required` implies that it does something more than set a suffix, imo.

Reply:
Two props do feel a bit much, but removing the default text would require consumers to provide the same text in many places.

Reply:
We ended up using two, very loosely connected, props. With the `optional` prop the default hint "niet verplicht" appears at the end of the label's text in parentheses, but only if no value for the `hint` prop was provided. If a `hint` is provided it will always display that value.